src/Dfc.CourseDirectory.Core/ReferenceData/Lars/LarsDataImporter.cs

```diff
@@ -187,21 +200,31 @@ Task ImportCategoryToSql()
 
             Task ImportLearnAimRefTypeToSql()
             {
-                var records = ReadCsv<UpsertLarsLearnAimRefTypesRecord>("LearnAimRefType.csv");
+                const string csv = "LearnAimRefType.csv";
+                var records = ReadCsv<UpsertLarsLearnAimRefTypesRecord>(csv).ToList();
+
+                static bool Excluded(UpsertLarsLearnAimRefTypesRecord r) => r.LearnAimRefTypeDesc.StartsWith("T Level", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
We’ve typically got local functions declared at the end of the scope rather than inline. 

Nit: naming (doesn’t read like a command).

Reply:
done :+1: 
thanks, that was helpful